scripts/ci/docker/zombienet_injected.Dockerfile

```
@@ -63,7 +63,7 @@ RUN mkdir -p /etc/zombie-net
 # USER nonroot
 
 # install rust
-ENV RUST_VERSION=1.85.0
+ENV RUST_VERSION=1.86.0
```

```suggestion
ENV RUST_VERSION=1.88.0
```

bump to 1.88 to use the same that ci-unified (https://hub.docker.com/layers/paritytech/ci-unified/latest/images/sha256-d447efa8eab4e0d6e1a72ed998d0e04b6b56a4320c1b1d8e9bbec285fe2f12c4)